### PR TITLE
Removed github Link & I18n Widget in footer

### PR DESCRIPTION
**Pull request by alianza**

### CODE REVIEW

**Comment by alianza** [comment]
⚡️ [Lighthouse report](https://storage.googleapis.com/lighthouse-infrastructure.appspot.com/reports/1625828500565-28171.report.html) for the changes in this PR:
| Category | Score |
| --- | --- |
| 🟢 Performance | 90 |
| 🟢 Accessibility | 100 |
| 🟢 Best practices | 100 |
| 🟢 SEO | 92 |
| 🟢 PWA | 92 |
 
*Lighthouse ran on [https://headless-commerce-poc-l9rqvyu8h-alianza.vercel.app/](https://headless-commerce-poc-l9rqvyu8h-alianza.vercel.app/)*

File: components/common/Footer/Footer.module.css [hunk 1]
```diff
@@ -1,4 +1,6 @@
 .link {
+  @apply cursor-pointer;
```
Comment (pascalw) [inline components/common/Footer/Footer.module.css:2]: Nitpick, but this shouldn't be needed. If something is a link, it should be an `a` tag which automatically has a pointer cursor. From a semantic and accessibility perspective It's bad practice to use elements other than `a` with a click handler, and make them look like a link.

**Review by pascalw — APPROVED** [review]

File: components/common/Footer/Footer.module.css (hunk 1 above)
Comment (alianza) [reply components/common/Footer/Footer.module.css]: Fixed this by turning it into a button! Now an outline does show when it's clicked, is that acceptable or should that be removed too?

Comment (pascalw) [reply components/common/Footer/Footer.module.css]: Perfect 👌 Indeed if something changes the state of the application, but it's not a link to a different page it should be a button.

**Pull request merged**